scss/_patterns_icons.scss

```diff
@@ -32,8 +32,6 @@ $social-icon-size: map-get($icon-sizes, heading-icon--small);
   @include vf-p-icon-instagram;
   @include vf-p-icon-linkedin;
   @include vf-p-icon-youtube;
-  @include vf-p-icon-canonical;
```

Comment:
We need to keep `@include vf-p-icon-canonical; @include vf-p-icon-ubuntu;` to make sure these icons are still part of Vanilla for backwards compatibility.

Reply:
Ah yes. Will add back in 👍